tests/test_dummy_converter.py

```diff
@@ -62,3 +51,55 @@ def test_default_geo_config_str() -> None:
     1    2    3
     1 1000    0
 """
+
+TEST_DIR = tempfile.TemporaryDirectory()
```

Comment:
Better to use pytest temporary dir fixture
https://docs.pytest.org/en/latest/how-to/tmp_path.html

Reply:
I have an issue with fixtures where they won't "unwrap" themselves and will pass the fixture function reference to the constructor, which would later throw an error.

Reply:
Refactored into a fixture and combined with the "assertless" test.